src/components/utils/rhf-inputs/directory-items-input.tsx

```diff
@@ -58,6 +60,10 @@ const DirectoryItemsInput: FunctionComponent<DirectoryItemsInputProps> = ({
     const types = useMemo(() => [elementType], [elementType]);
     const [directoryItemSelectorOpen, setDirectoryItemSelectorOpen] =
         useState(false);
+
+    const [selectedChip, setSelectedChip] = useState<any[] | undefined>([]);
+    const [expandedNodes, setExpandedNodes] = useState<any[] | undefined>([]);
```

Comment:
I don't think they could be undefined
Replace 'any' with a specific type if possible